app/src/main/java/io/scalac/degree/android/fragment/TalksFragment.java

```diff
@@ -72,14 +73,17 @@
     @ColorRes(R.color.scheduled_not_star_color)
     int notscheduledStarColor;
 
+    @ViewById(R.id.listView1)
+    ListView listView;
+
+    @ColorRes(R.color.primary_text_45)
```

Comment:
To chodzi o alpha channel, opacity, więcej tutaj, https://www.google.com/design/spec/style/color.html#color-ui-color-application